service/src/main/kotlin/no/nav/su/se/bakover/service/sak/SakService.kt

```diff
@@ -1,17 +1,27 @@
 package no.nav.su.se.bakover.service.sak
 
 import arrow.core.Either
+import no.nav.su.se.bakover.common.periode.Periode
 import no.nav.su.se.bakover.domain.Fnr
 import no.nav.su.se.bakover.domain.NySak
 import no.nav.su.se.bakover.domain.Sak
 import no.nav.su.se.bakover.domain.Saksnummer
+import no.nav.su.se.bakover.domain.revurdering.GjeldendeVedtaksdata
+import java.time.LocalDate
 import java.util.UUID
 
 interface SakService {
     fun hentSak(sakId: UUID): Either<FantIkkeSak, Sak>
     fun hentSak(fnr: Fnr): Either<FantIkkeSak, Sak>
     fun hentSak(saksnummer: Saksnummer): Either<FantIkkeSak, Sak>
     fun opprettSak(sak: NySak)
+    fun kopierGjeldendeVedtaksdata(sakId: UUID, fraOgMed: LocalDate): Either<KunneIkkeKopiereGjeldendeVedtaksdata, GjeldendeVedtaksdata>
```

Comment:
Denne føles litt malplassert. Hvorfor ikke ha den i en VedtakService e.l.? Og hvorfor ligger GjeldendeVedtaksdata under revurdering og ikke vedtak?

Reply:
Klarte ikke helt å bestemme meg for hvor den skulle ligge.. endte opp med sak siden det er en sak-id som sendes inn for å generere vedtakshistorikk for hele saken. Åpen for å flytte den hvis det er forvirrende?

Reply:
Jeg tenkte bare at kopierGjeldendeVedtaksdata er nærmere vedtak enn sak.

Reply:
Fair. Flytter.